socrates/lib/subscribers/subscriber.js

```diff
@@ -36,7 +36,15 @@ class Subscriber {
   }
 
   livesInGermany() {
-    return this.country() === 'DE';
+    return this.state.country && this.state.country.length > 0 && this.country() === 'DE';
+  }
+
+  isDiversity() {
+    return this.addon().ladiesTShirt() || this.hasNonMalePronoun() || !this.livesInGermany();
+  }
+
+  hasNonMalePronoun() {
```

Comment:
Diese Funktion wird doch nirgends benutzt außer in Zeile 43, oder? Bitte inlinen und entfernen.